Sources/Diagnostics/DiagnosticsFileHandler.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DiagnosticsFileHandler.swift
+//
+//  Created by Nacho Soto on 6/16/23.
```

Comment:
lol
I love that Nacho is still adding RC code

Reply:
👀